libs/perun/components/src/lib/group-menu/group-menu.component.html

```diff
@@ -1,13 +1,57 @@
-<button (onMenuClose)="whenClosed.emit()" [disabled]="disabled" [mat-menu-trigger-for]="groupMenu" mat-icon-button>
-  <mat-icon>more_vert</mat-icon>
-</button>
-<mat-menu #groupMenu="matMenu" >
-  <button mat-menu-item (click)="onMoveGroup()">
+<div *ngIf="displayButtons" class="buttons-container">
+  <button [cdkCopyToClipboard]="group.name" [matTooltipPosition]="'above'" class="ml-auto" color="accent"
```

Comment:
I don't think there is any reason why this button should be more visible than the others. I would just make it black as well.